common/service/dynamicconfig/config.go

```diff
@@ -260,6 +263,37 @@ func (c *Collection) GetFloat64PropertyFilteredByShardID(key Key, defaultValue f
 	}
 }
 
+// GetFloatPropertyFilteredByTaskQueueInfo gets property with taskQueueInfo as filters and asserts that it's an integer
+func (c *Collection) GetFloatPropertyFilteredByTaskQueueInfo(key Key, defaultValue float64) FloatPropertyFnWithTaskQueueInfoFilters {
+	return func(namespace string, taskQueue string, taskType enumspb.TaskQueueType) float64 {
+		val := defaultValue
+		var err error
+
+		filterMaps := []map[Filter]interface{}{
+			getFilterMap(NamespaceFilter(namespace), TaskQueueFilter(taskQueue), TaskTypeFilter(taskType)),
+			getFilterMap(NamespaceFilter(namespace), TaskQueueFilter(taskQueue)),
+		}
+
+		for _, filterMap := range filterMaps {
+			val, err = c.client.GetFloatValue(
+				key,
+				filterMap,
+				defaultValue,
+			)
+			if err != nil {
+				c.logError(key, err)
+			}
+
+			if val != defaultValue {
```

Comment:
fyi, this is float64, right? equals/not equals could lead to some weird behavior depending on precision

Reply:
guess for dynamic config, this is fine. there should be no N/A case, or precision related case